app/helpers/automated_tests_helper.rb

```diff
@@ -1,6 +1,11 @@
 # Helper methods for Testing Framework forms
 module AutomatedTestsHelper
 
+# Prototype code that needs to be removed:
+# Methods: add_test_file_link, add_lib_file_link, add_parser_file_link
+# $F, .insert() and .replace() are all Prototype functions
+# Any place with $('id') will need to be replaced with something like jQuery('#id')
```

Comment:
Line is too long. [83/80]<br>Incorrect indentation detected (column 0 instead of 2).